src/cargo/ops/cargo_remove.rs

```diff
@@ -22,7 +23,7 @@ pub struct RemoveOptions<'a> {
 }
 
 /// Remove dependencies from a manifest
-pub fn remove(options: &RemoveOptions<'_>) -> CargoResult<()> {
+pub fn remove(workspace: &Workspace<'_>, options: &RemoveOptions<'_>) -> CargoResult<()> {
```

Comment:
Should `Workspace` be added to `RemoveOptions` instead?

Reply:
That would certainly make more sense to me. I had pretty much just copied the function signature from cargo add and ended up with this.